src/errorHandler.js

```diff
@@ -0,0 +1,47 @@
+import SourceFetcher from './sourceFetch.js';
+
+export default class errorPopupHandler {
```

Comment:
1. please rename to `ErrorPopupHandler` (class name usually starts with an upper case char)

2. let's split this class into two ones (both in separate files):


-  `ErrorPopup` a class that is responsible for closing/opening a pop-up:

``` js
class ErrorPopup {
  open = (message) => { /* ... */ }; // no need to pass error object here - just a message string
  close = () => { /* ... */ };
}
```
- `ErrorPopupHandler` - the class that provides a lazy-loaded singleton instance of `ErrorPopup` (imagine that `ErrorPopup` is third-party library and you cannot modify it, but there is technical requirement to have only one instance throughout the application):
``` js
class ErrorPopupHandler {
  static getInstance = async () => { /* ... */ }; // 1. use async/await; 2) lazy-load `ErrorPopup` module and instantiate its instance (remove `lazyload.js` - it does not meet the task requirement)
}
```

3. try to use string literals and `innerHTML` instaed of `createElement/setAttribute` - this is more readable + looks like a template in Angular (or JSX in React)